activityMonitor.js

```diff
@@ -0,0 +1,43 @@
+const childProcess = require("child_process");
+const os = require("os");
+const fs = require("fs");
+
+const WINDOWS_COMMAND = `powershell "Get-Process | Sort-Object CPU -Descending | Select-Object -Property Name, CPU, WorkingSet -First 1 | ForEach-Object { $_.Name + ' ' + $_.CPU + ' ' + $_.WorkingSet }"`;
+const UNIX_COMMAND = "ps -A -o %cpu,%mem,comm | sort -nr | head -n 1";
+
+const isWindows = os.type() === "Windows_NT";
+const command = isWindows ? WINDOWS_COMMAND : UNIX_COMMAND;
+
+setInterval(
+  () =>
+    childProcess.exec(command, (error, result) => {
+      console.clear();
```

Comment:
`console.clear` and `console.log` indeed give us almost same result. But I would rather prefer to check and use :
- `process.stdout.clearLine()` instead of console.clear. `console.clear` clears whole console. The stdout clears just the line. And if you use the clearLine you should combine it with the `process.stdout.cursorTo(0);` to move the cursor at beginning. 
- ` process.stdout.write()` instead of `console.log()`. By default console.log adds a new line at the end. But stdout.write continue on the same line.

Reply:
@fedaipaca, thank you for the great explanation! The result looks much better with process.stdout!